src/app/archive/zkapp_tables.sql

```diff
@@ -110,7 +115,7 @@ CREATE TABLE zkapp_nonce_bounds
 CREATE TYPE zkapp_account_precondition_type AS ENUM ('full', 'nonce', 'accept');
 
 /* NULL convention */
-CREATE TABLE zkapp_account
+CREATE TABLE zkapp_precondition_accounts
```

Comment:
the names are confusing- `zkapp_precondition_accounts` and `zkapp_account_precondition`. 
How about `account_precondition_values` or `account_precondition_terms` ?

Reply:
I think I want to keep `zkapp_precondition_accounts`, it mirrors the OCaml type `Zkapp_precondition.Account.t`, and has the same contents.

The type `zkapp_account_precondition_type` can be `zkapp_precondition_type`.

Reply:
But let me look again, the code has changed ...

Reply:
Yes, I think my suggestions reflect the current code.

Reply:
I don't think we need to strictly mirror OCaml types (and we don't in many cases). I'd find the names confusing if I didn't know the OCaml types, but that's just me. I'll let @bkase or someone from product weigh in on the ux of this

Reply:
Yes I agree with @deepthiskumar -- we should be optimizing for people who aren't familiar with the code.

Having said that, it's worth putting a comment in the schema wherever the table name diverges from the code (so in this case we can add a comment saying this corresponds with `Zkapp_precondition.Account.t`